packages/data-point/data-point.d.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
shouldn't this file be renamed `index.d.ts` ?

Reply:
done